### ISSUE TITLE
Target .NET Core 2.0 instead of .NET 4.5

### ISSUE THREAD

**Issue by sparr** [rallion/depressurizer#87]
If this project was re-targeted against .NET Core then it would run natively on OSX and Linux and generally be more portable.

This would resolve #54

**Comment by mvegter** [comment]
@sparr Not really. As far as I'm aware .NET Core doesn't have any (G)UI support as it is ["focused on services and command line applications"](https://github.com/dotnet/core/issues/374#issuecomment-336340517). I'm currently working in [my fork](https://github.com/mvegter/Depressurizer) on this support, it is a [very low priority](https://github.com/mvegter/Depressurizer/issues/14).

Best option is to stay on the .NET framework and using WinForms, as the [mono-project](http://www.mono-project.com/docs/gui/winforms/) does support WinForms.

**Comment by sparr** [comment]
Thanks. I had not realized there was a GUI.

**Comment by kaosine** [comment]
I just had someone tell me about this project and wondered about linux availibilty. I swear I saw somewhere that WPF got ported but I'd have to find it again. I'd love to have it availible if it's possible and I guess I'll do some research. (sorry for misspellings, having some issues with right clicking...)

**Comment by GrantByrne** [comment]
.net core 3 has wpf support; however, it only runs in Windows. 

I think it has to do with WPF relying on a Windows only graphics framework.

**Comment by mvegter** [comment]
@darsparx and @holymoo Depressurizer uses WinForms not WPF, thus .NET Core doesn't really help.
